Popup in compact interactive list (#2149)
cerner/terra-framework: packages/terra-popup/CHANGELOG.md

## Unreleased

* Changed

>   * Added event handlers to call `stopPropagation()` on key and focus events on PopupContent for `terra-popup` support in `terra-compact-interactive-list` cell.

Reviewer: can this feature be used outside of compact interactive list?

Author: It works everywhere, doesn't matter inside of CIL or outside of it, as outside of the compact interactive list it makes no difference. I just wanted to provide a reason why it was added, as it makes no difference for all the cases when popoup is outside of CIL (or any other component that has key navigation).

Reviewer: Then I would reword this to be something generic like:

```suggestion
  * Added event handlers to call `stopPropagation()` on key and focus events on PopupContent in other components.
```
